make/test.mk

```diff
@@ -96,16 +95,10 @@ print-operator-logs:
 .PHONY: login-as-admin
 login-as-admin:
 ifeq ($(OPENSHIFT_BUILD_NAMESPACE),)
-    ifeq ($(IS_CRC),)
-        ifneq ($(IS_OS_3),)
-        	# is running locally and against OS 3, so we assume that it's minishift
-			$(info logging as system:admin")
-			oc login -u system:admin 1>/dev/null
-        endif
-    else
+    ifneq ($(IS_CRC),)
```

Comment:
I would just drop `login-as-admin` completely. We expect the user to be logged into cluster as an admin no matter what cluster it is. No special login treatment for CRC. Just to simplify the flow.